pkgs/dart_services/lib/server.dart

```diff
@@ -21,10 +21,12 @@ final Logger _logger = Logger('services');
 
 Future<void> main(List<String> args) async {
   final parser = ArgParser()
-    ..addOption('channel',
-        valueHelp: 'channel', help: 'The SDK channel (required).')
     ..addOption('port', valueHelp: 'port', help: 'The port to listen on.')
     ..addOption('redis-url', valueHelp: 'url', help: 'The redis server url.')
+    ..addOption('storage-bucket',
+        valueHelp: 'name',
+        help: 'The name of the Cloud Storage bucket for compilation artifacts.',
+        defaultsTo: 'nnb_artifacts')
```

Comment:
```suggestion
        defaultsTo: 'nnbd_artifacts')
```

Reply:
Do we need this to be a configurable cli param?

Reply:
I found that I needed it because I am running this on a separate GCP project for testing.

Reply:
I also think we should go back to the compilation_artifacts bucket and clean up some of the old buckets.